docs/FAQ.zh-CN.md

```diff
@@ -0,0 +1,74 @@
+<!--
```

Comment:
We don't have docs in Chinese anymore, please sync the latest codes from master.

Reply:
removed.